src/nrc/datamodel/models.py

```diff
@@ -141,13 +142,24 @@ class Notificatie(models.Model):
     forwarded_msg = models.JSONField(encoder=DjangoJSONEncoder)
     kanaal = models.ForeignKey(Kanaal, on_delete=models.CASCADE)
 
+    @property
+    def last_attempt(self):
+        return (
+            self.notificatieresponse_set.aggregate(Max("attempt"))["attempt__max"] or 0
+        )
+
     def __str__(self) -> str:
         return "Notificatie ({})".format(self.kanaal)
 
 
 class NotificatieResponse(models.Model):
     notificatie = models.ForeignKey(Notificatie, on_delete=models.CASCADE)
     abonnement = models.ForeignKey(Abonnement, on_delete=models.CASCADE)
+    attempt = models.PositiveSmallIntegerField(
+        default=1,
+        verbose_name=_("attempt"),
+        help_text=_("Indicates to which iteration of retrying this response belongs"),
```

Comment:
```suggestion
        help_text=_("Indicates to which iteration of retrying this response belongs."),
```
not entirely correct though - the first retry is the second delivery attempt. I'd phrase it more like "Indicates to which delivery attempt this response belongs."